security/conf/registration.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Is JSON mandatory? Could we use YAML?

Reply:
No, according to documentation `spire-server` doesn't support YAML:
```
Path to a file containing registration data in JSON format (optional).
```